### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by vasusharma7**

### CODE REVIEW

File: backend/config/db.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 const config = require('config');
-const db = config.get('mongoURI');
-// const db = config.get('mongoLocal');
+// const db = config.get('mongoURI');
+const db = config.get('mongoLocal');
```
Comment (Pranav2612000) [inline backend/config/db.js:4]: This was made by you to use local mongo right? Could you revert the changes made to work with local mongo?

Comment (naman-modi) [reply backend/config/db.js:4]: This PR still isn't merged, right?
@Pranav2612000 @vasusharma7 

It only solves 'NULL' issue or has any other fixes?

**Comment by naman-modi** [comment]
user name 'null' issue fixed in PR #108 
@vasusharma7 Any other issues fixed in this?

If not, close this PR.
If yes, please reflect in merge-webcam branch and effectively in PR #108

File: backend/config/db.js (hunk 1 above)
Comment (vasusharma7) [reply backend/config/db.js:4]: It also solves, redirect to login issue for guest.

Comment (vasusharma7) [reply backend/config/db.js:4]: > This was made by you to use local mongo right? Could you revert the changes made to work with local mongo?

yes, I will do

**Comment by Pranav2612000** [comment]
Does this need to be merged?

**Comment by savannahar68** [comment]
I guess other "merged" PRs have covered this PR. We can close this.

Right @vasusharma7 ?

**Comment by vasusharma7** [comment]
Yes. All the issues are solved in my other PRs.
 Can safely close this.
